server/migrations/20190511074745_create_notifications_table.js

```diff
@@ -0,0 +1,13 @@
+exports.up = knex =>
```

Comment:
There should be a field called `read_at` so a user can mark a notification as read.

Reply:
implemented